tests/unit/commands/migration/test_migration.py

```diff
@@ -29,7 +29,8 @@
 class TestMigration(unittest.TestCase):
```

Comment:
Don't we need to verify we call self.api.Users(self.login).Migrations(doArgs.id).Delete() at some point?

Reply:
Yes It's a good idea, I verify the execution in the two last tests.